types/index.d.ts

```diff
@@ -45,6 +45,10 @@ declare namespace fastifySwaggerUi {
      * @default /documentation
      */
     routePrefix?: string;
+    /**
+     * Add an index prefix. This is for when the Fastify server is behind path based routing.  ex. NGINX
+     */
+    indexPrefix?: string;
```

Comment:
Can you add a test for this type? We use tsd.

Reply:
I'm not sure what you mean.  There is a test in [test/route.test.js,](https://github.com/fastify/fastify-swagger-ui/pull/186/files#diff-71868d2908a96fcd854d3015e6275d3841daa70478fa84f46661f23da563c802R585-R616)   Is there something else you are looking for?

Reply:
Hi @DavidTanner! 👋🏼 
You can see an example [here](https://github.com/fastify/fastify-swagger-ui/blob/master/types/types.test-d.ts). Basically, by running the `tsd` tool, you can run tests for the TS types. You can then check locally by running `npm run typescript`, (which [will run](https://github.com/fastify/fastify-swagger-ui/blob/master/package.json#L18) `tsd`)

Reply:
Happy to push the TS test update if this can help to unlock the release of this PR 😉

Reply:
https://github.com/fastify/fastify-swagger-ui/pull/189/commits